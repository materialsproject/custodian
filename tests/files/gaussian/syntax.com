%chk=Optimization.chk
%mem=100GB
%NProcShared=28
#P B3LYP/6-311++G** Opt=(MaxCycless=100) SCF SCRF=(PCM, Solvent=TetraHydroFuran)

H24 O12

0 1
O -0.535940 2.412480 0.000000
H 0.434060 2.412480 0.000000
H -0.859270 1.623360 -0.462230
O -0.477040 0.515290 0.000000
H 0.492960 0.515290 0.000000
H -0.800370 -0.331850 0.344550
O 1.941420 1.473710 -0.115560
H 2.911420 1.473710 -0.115560
H 1.618090 0.650870 -0.514660
O 1.506410 -2.739190 -0.033320
H 2.476410 -2.739190 -0.033320
H 1.183080 -2.953640 0.855710
O -2.481410 -1.412410 -0.089680
H -1.511410 -1.412410 -0.089680
H -2.804740 -1.796910 0.740090
O 0.267510 -1.375730 -0.089680
H 1.237510 -1.375730 -0.089680
H -0.055820 -1.963460 -0.790340
O 2.079230 3.193980 0.024790
H 3.049230 3.193980 0.024790
H 1.755900 3.501110 -0.836620
O -2.729610 0.744610 0.024790
H -1.759610 0.744610 0.024790
H -3.052940 0.984940 -0.857590
O -2.260190 4.168690 -0.056620
H -1.290190 4.168690 -0.056620
H -2.583520 4.985440 -0.468050
O -2.762450 3.169940 -0.056620
H -1.792450 3.169940 -0.056620
H -3.085780 2.467980 0.529570
O 0.687880 4.885180 -0.056620
H 1.657880 4.885180 -0.056620
H 0.364550 5.613770 -0.609360
O -1.168310 -3.669310 -0.056620
H -0.198310 -3.669310 -0.056620
H -1.491640 -3.998110 -0.909990



