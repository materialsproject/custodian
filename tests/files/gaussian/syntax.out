 Entering Gaussian System, Link 0=g16
 Initial command:
 /gpfs/software/gaussian/g16/l1.exe "/gpfs/scratch/ratwi/tests/Custodian/syntax/formbx/Optimization/Gau-14487.inp" -scrdir="/gpfs/scratch/ratwi/tests/Custodian/syntax/formbx/Optimization/"
 Entering Link 1 = /gpfs/software/gaussian/g16/l1.exe PID=     14488.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                18-Mar-2024 
 ******************************************
 %chk=Optimization.chk
 %mem=100GB
 %NProcShared=28
 Will use up to   28 processors via shared memory.
 ----------------------------------------------------------------------
 #P B3LYP/6-311++G** Opt=(MaxCycless=100) SCF SCRF=(PCM, Solvent=TetraH
 ydroFuran)
 ----------------------------------------------------------------------
 QPErr --- A syntax error was detected in the input line.
 #P B3LYP/6-311++G** Opt=(MaxCycless=100)
                          '
 Last state= "OPT2"
 TCursr=    143767 LCursr=        25
 Error termination via Lnk1e in /gpfs/software/gaussian/g16/l1.exe at Mon Mar 18 16:34:23 2024.
 Job cpu time:       0 days  0 hours  0 minutes  3.3 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  0.1 seconds.
 File lengths (MBytes):  RWF=    112 Int=      0 D2E=      0 Chk=     16 Scr=     16
